mkdocs.yml

```diff
@@ -154,8 +154,8 @@ markdown_extensions:
   - attr_list
   - md_in_html
   - pymdownx.emoji:
-      emoji_index: !!python/name:materialx.emoji.twemoji
-      emoji_generator: !!python/name:materialx.emoji.to_svg
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
```

Comment:
What is this change for?

Reply:
Accommodating upstream changes. I synced our [mkdocs-material-insiders](https://github.com/kolenaIO/mkdocs-material-insiders) fork this morning and it spits out a ton of warnings when using the old `materialx` import.